### PR TITLE
return mutated object to avoid on query

### PR DESCRIPTION
**Pull request by henryzbw**
Only made change for bet now
want to make sure it's the way you want
and then continue to others

don't need to change server now

### CODE REVIEW

File: src/stores/TransactionStore.js [hunk 1]
```diff
@@ -558,7 +558,7 @@ export default class TransactionStore {
           amount,
         });
 
-        await this.onTxExecuted(index, tx);
+        await this.onTxExecuted(index, tx, new Transaction(createBet));
```
Comment (dwalintukan) [inline src/stores/TransactionStore.js]: let's make a change to `mutations.js` where `createTransaction()` returns a `new Transaction()` instance.

**Comment by henryzbw** [comment]
@dwalintukan verified it works. please have a look at the code after your last review. If it's okay, i will ship it.

File: src/stores/TransactionStore.js [hunk 2]
```diff
@@ -954,13 +954,12 @@ export default class TransactionStore {
   @action
   executeTransfer = async (index, tx) => {
     try {
-      const { data: { transfer } } = await createTransaction('transfer', {
+      const newTx = await createTransaction('transfer', {
```
Comment (dwalintukan) [inline src/stores/TransactionStore.js]: nit: lol why is this one named different than the other ones?

File: src/network/graphql/mutations.js [hunk 3]
```diff
@@ -66,6 +67,8 @@ class GraphMutation {
  * @param {string} mutationName Name of the mutation.
  * @param {object} args Arguments for the mutation.
  */
-export function createTransaction(mutationName, args) {
-  return new GraphMutation(mutationName, args, TYPE.transaction).execute();
+export async function createTransaction(mutationName, args) {
+  const res = await new GraphMutation(mutationName, args, TYPE.transaction).execute();
+  const { data: { [mutationName]: ret } } = res;
```
Comment (dwalintukan) [inline src/network/graphql/mutations.js]: nit: `[mutationName]: tx`

**Pull request merged**